### DISCUSSION TITLE
Duplicar repositorio

### DISCUSSION THREAD

**Discussion by barucsi** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#10] [Q&A] [answered]
Hola! actualmente las katas en perfil de GitHub con la cuenta de launchX. Quisiera saber si existe la forma de tener el repositorio aquí y en mi perfil escolar pero de modo que no sea un fork?

Nota: Mire la opcion de transferir pero quiero transferirlo por temor a que cuando revisen mis Katas no las encuentren por haber transferido el repo a otro perfil.

Saludos y gracias

**Comment by Ingrzs** [comment]
Aprovecho para preguntar, intento hacer fork y me dice que necesito permisos de la organización,pero veo que ya mas de 100 personas hicieron fork...o hicieron pull request? 🤔

**Reply by doguedogue** [reply]
haz el fork directamente del repo del commander.

**Comment by doguedogue** [comment, marked as answer]
- claro, puedes clonar un repositorio y hacerlo independiente (darle los créditos/verificar licencias).
- Puedes crear un repositorio tuyo.
- Puedes direccionar el repositorio clonado a un repositorio tuyo.

**Reply by barucsi** [reply]
Pero se clonan los commits y toda la info o sería como un nuevo commit?

**Reply by doguedogue** [reply]
- Clonas el repositorio de la organización para tenerlo en tu local.
- Subes únicamente tus ejercicios en tu repositorio nuevo, y puedes hacerle todos los commits que gustes a tu ritmo.
